projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -497,6 +443,73 @@
 	}
 }
 
+.jetpack-field-multiple:where(.is-style-button) {
+	// Reduced specificity to 0,1,0 so that global styles can override it.
+	:where(.jetpack-field-option) {
+		background: var(--jetpack--contact-form--button-outline--background-color);
+		color: var(--jetpack--contact-form--button-outline--text-color);
+		border: var(--jetpack--contact-form--button-outline--border);
+		border-width: var(--jetpack--contact-form--button-outline--border-size);
+		border-color: currentColor;
+		border-radius: var(--jetpack--contact-form--button-outline--border-radius);
+		font-size: var(--jetpack--contact-form--font-size, 16px );
+		font-family: var(--jetpack--contact-form--font-family);
+		padding: var(--jetpack--contact-form--button-outline--padding);
+		line-height: var(--jetpack--contact-form--button-outline--line-height);
+	}
+
+	// Increase specificity to override the `align-items: baseline` from the `jetpack-field-option` class.
+	.jetpack-field-option.jetpack-field-option {
+		align-items: center;
+	}
+
+	.jetpack-field-option.field-option-radio {
+		// TODO - Investigate if this is needed.
+		position: relative;
+		flex-direction: row-reverse;
+		gap: 16px;
+
+		// Hide the radio input when in the button style.
+		// TODO - Check why this isn't done for the checkboxes.
+		.jetpack-option__type {
+			display: none;
+		}
+	}
+
+	.jetpack-field-option .jetpack-option__type {
+		color: inherit;
+	}
+
+	.wp-block-jetpack-field-option-radio {
+
+		.jetpack-option__type {
+			display: none;
+		}
+	}
+}
+
+// TODO - See if we can find usage of this class. It's possibly a dead style.
```

Comment:
There are a few comments like this I've added so hopefully we can double-check these later and remove any dead css.